server/controllers/userVacancyController.js

```diff
@@ -26,6 +26,6 @@ exports.postUserVacancy = (req, res) => {
         user: req.body.userId,
         vacancy: req.body.vacancyId
     })
-        .then(userVacancies => res.json(userVacancies))
+        .then(userVacancies => res.status(201).json(userVacancies))
```

Comment:
Forceer je hiermee niet dat je altijd een succesvolle enrollment in vancancies, want die icons check toch op de status code

Reply:
Hoe zou ik dan een code terug moet geven die gecheckt kan worden?

Reply:
Never mind, code klopt volgens mij. Ik dacht dat je altijd 201 krijgt ongeacht het resultaat waardoor je een succesvol enrollment krijgt, maar als het goed regelt de catch dat al.